src/multivariate/mvnormal.jl

```diff
@@ -26,30 +26,30 @@ abstract AbstractMvNormal <: ContinuousMultivariateDistribution
 
 ### Generic methods (for all AbstractMvNormal subtypes)
 
-insupport(d::AbstractMvNormal, x::AbstractVector) =
+insupport{T<:Real}(d::AbstractMvNormal, x::AbstractVector{T}) =
     length(d) == length(x) && allfinite(x)
 
 mode(d::AbstractMvNormal) = mean(d)
 modes(d::AbstractMvNormal) = [mean(d)]
 
-entropy(d::AbstractMvNormal) = (length(d) * (Float64(log2π) + 1) + logdetcov(d))/2
+entropy(d::AbstractMvNormal) = 0.5 * (length(d) * (Float64(log2π) + 1.0) + logdetcov(d))
```

Comment:
Sorry, these are changes we'd already made on master. Will push correct versions up momentarily.